code/modules/power/supermatter/supermatter.dm

```diff
@@ -212,6 +216,9 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	AddComponent(/datum/component/supermatter_crystal, CALLBACK(src, PROC_REF(wrench_act_callback)), CALLBACK(src, PROC_REF(consume_callback)))
 	soundloop = new(src, TRUE)
 
+	if(!isnull(check_holidays(FESTIVE_SEASON)) && istype(src, /obj/machinery/power/supermatter_crystal/engine))
+		holiday_lights()
```

Comment:
If you're only going to allow this on the `/engine` subtype, then why is it here and not in /engine/Initialize?

Reply:
Yeah I don't see why it needs to be restricted to the engine.